recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - click >=6.6
     - cloudpickle >=1.5.0
     - cytoolz >=0.8.2
-    - dask-core {{ version }}
+    - dask-core =={{ version }}
```

Comment:
Same concern as noted in comment ( https://github.com/conda-forge/dask-feedstock/pull/190/files#r941655804 ). Though we can discuss in that thread (instead of here) for simplicity.